src/onyx/tasks/s3.clj

```diff
@@ -28,7 +29,9 @@
   ([task-name :- s/Keyword
     bucket :- s/Str
     serializer-fn :- os/NamespacedKeyword
-    task-opts :- {s/Any s/Any}]
+    task-opts :- {s/Any s/Any}
+    encryption :- {(s/optional-key :aes256) s/Keyword}]
```

Comment:
`:aes256` should be inferred from a list of allowed encryption schemes, shared with the conditional above.